### ISSUE TITLE
CuDNN Error

### ISSUE THREAD

**Issue by taylover-pei** [NVlabs/PWC-Net#37]
Congratulations! You have done a great job.
I have a problem when I execute the code:
python script_pwc.py './data/frame_0010.png' './data/frame_0011.png' './tmp/frame_0010.flo'
and then, there is a problem just show here:
 
![image](https://user-images.githubusercontent.com/27360803/45813193-85f8e400-bd04-11e8-8093-8856e8120040.png)
I download the pytorch use the code: conda install pytorch=0.2 torchvision cuda80 -c pytorch
and correctly install the cuDNN。So, I‘m so worried about this. Why does this happen? How can I solve it.
Looking forward to your reply.

**Comment by nankeermeng** [comment]
@taylover-pei I met the same problem, do you solve it ?

**Comment by taylover-pei** [comment]
No, I did not. I have no idea. I just try another one, just like this:https://github.com/sniklaus/pytorch-pwc

**Comment by taylover-pei** [comment]
> @taylover-pei I met the same problem, do you solve it ?

No, I did not. I have no idea. I just try another one, just like this:https://github.com/sniklaus/pytorch-pwc

**Comment by luvegood** [comment]
@taylover-pei @nankeermeng  Sorry for the late reply, I don't know why I missed this thread.
1. Double check if you are using Python 2.7 & PyTorch 0.2 & CUDA 8.0, 
2. were you able to at least get pytorch to work? (e.g running some simple official training examples)
